includes/blocks/class-sensei-ct-blocks.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * File containing the class Sensei_CT_Blocks.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_CT_Blocks
+ */
+class Sensei_CT_Blocks extends Sensei_Blocks_Initializer {
```

Comment:
I think it'd be more readable to ditch the abbreviations and use full words everywhere (unless it's something very common across Wordpress/Sensei). Same for the CSS classes with `pnl`.

For the PHP classes, we can also use a namespaces (unless they break the class autoloader in Sensei).

Reply:
Agree. Wasn't sure about the abbreviations. I think namespaces would be a correct way to go. Ideally we could use PSR-0/4. But we gotta follow WP conventions with our file names.

- Updated the class names to use namespaces and it is more readable now.
- Updated the html classes with more descriptive names too.